src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -37,32 +52,19 @@ func Configure(cfg *config.Config, serverSession bool, newClient *http.Client) {
 		},
 	}
 
-	if cfg == nil {
-		analytics.SessionEvent(httpClient, sessionPayload)
-		return
-	}
-
-	if cfg.InstanceId != "" {
-		sessionPayload.ClientId = cfg.InstanceId
-	}
-	if cfg.Version != 0 {
-		sessionPayload.ClientVersion = string(cfg.Version)
-	}
-
-	if serverSession {
-		sessionPayload.Hostname = cfg.Server.RegisterAt
-	} else {
-		sessionPayload.Hostname = "localhost"
```

Comment:
I included this originally because I thought it might be convenient to store server-side analytics at some point. Taking it out though since it's questionable we will need it.